resource_estimator/src/estimates/modeling/logical_qubit.rs

```diff
@@ -30,16 +27,12 @@ pub struct LogicalQubit<P: TPhysicalQubit> {
     logical_error_rate: f64,
 }
 
-impl<P: TPhysicalQubit> LogicalQubit<P> {
+impl<P> LogicalQubit<P> {
     pub fn new(
-        ftp: &impl ErrorCorrection<PhysicalQubit = P>,
+        ftp: &impl ErrorCorrection<Qubit = P>,
```

Comment:
to confirm that we want to use Qubit for PhysicalQubit and omit Physical

Reply:
Yes, it's more generic. I can envision situations in which error correction is on top of logical qubits.